web_development_101/project_html_css.md

```diff
@@ -10,13 +10,13 @@ Using the browser's developer tools (right clicking something on the page and cl
 These skills will be helpful for you when you start building.  Either try them yourself or at least make sure you know how to do it:
 
 1. Two ways to move a div around on the page
-1. Stick a div onto the bottom or top of the page
-1. Identify the background color of an existing webpage
-1. Grab the URL for an image from an existing webpage
-1. Center an element horizontally
-1. Identify three ways you can include your CSS styles in a page
-1. Understand how to use classes and ids to target CSS at specific elements on the page
-1. Build a very basic form (even if it doesn't "go" anywhere)
+2. Stick a div onto the bottom or top of the page
+3. Identify the background color of an existing webpage
+4. Grab the URL for an image from an existing webpage
+5. Center an element horizontally
+6. Identify three ways you can include your CSS styles in a page
+7. Understand how to use classes and ids to target CSS at specific elements on the page
+8. Build a very basic form (even if it doesn't "go" anywhere)
```

Comment:
This can be left as is. It is valid markdown to have an ordered list like this...

Reply:
It is valid markdown, but all the other ordered lists on the page are numbered correctly.